src/AWS.Deploy.Recipes/CdkTemplates/ConsoleAppECSFargateService/Generated/Configurations/AutoScalingConfiguration.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+// This is a generated file from the original deployment recipe. It contains properties for
+// all of the settings defined in the recipe file. It is recommended to not modify this file in order
+// to allow easy updates to the file when the original recipe that this project was created from has updates.
+// This class is marked as a partial class. If you add new settings to the recipe file, those settings should be
+// added to partial versions of this class outside of the Generated folder for example in the Configuration folder.
+
+namespace ConsoleAppEcsFargateService.Configurations
+{
+    public partial class AutoScalingConfiguration
+    {
+        const int defaultCooldown = 300;
+
+        public bool Enabled { get; set; }
+
+        public int MinCapacity { get; set; } = 3;
+
+        public int MaxCapacity { get; set; } = 6;
+
+        public enum ScalingTypeEnum { Cpu, Memory }
+
+        public ScalingTypeEnum? ScalingType { get; set; }
+
+
+
+        public double CpuTypeTargetUtilizationPercent { get; set; } = 70;
+
+        public int CpuTypeScaleInCooldownSeconds { get; set; } = defaultCooldown;
+
+        public int CpuTypeScaleOutCooldownSeconds { get; set; } = defaultCooldown;
```

Comment:
```suggestion
        public CpuTypeScaling? CpuTypeScaling { get; set; }

```

```
    public class CpuTypeScaling
    {
        public double TargetUtilizationPercent { get; set; } = 70;
        public int ScaleInCooldownSeconds { get; set; } = 300;
        public int ScaleOutCooldownSeconds { get; set; } = 300;
    }
```

Reply:
Since the nesting of the configuration parameters map 1 to 1 to the option settings I don't want to create another level of option settings in the recipe file. So I want to keep the settings flat here.

Reply:
Fair enough.